opengovsg/plumber
# PLU-70: [TILES-13] frontend: allow side scrolling

## Styling changes to allow for side-scrolling

Right now, the widths of each column are equal, and it will be too narrow for more than 4/5 columns. So side-scrolling is required for that and to allow adjustable widths.

**How to test**
1. in `column-helper.tsx`, set the minSize to a larger number like 700. 
2. Refresh page and scroll sideways 
3. Check adding new rows work with the new scroll


https://github.com/opengovsg/plumber/assets/10072985/3ed6df9d-dcd4-45ce-a065-3d45c336b246
